apps/frontend/components/FormValidation/FormValidationHeader.tsx

```diff
@@ -121,18 +120,6 @@ const FormValidationHeader = ({
         justifyContent="space-between"
       >
         <Stack direction="row" gap={theme.spacing(5)}>
-          <Controller
```

Comment:
I think we should keep this info somewhere but maybe not as an editable field indeed.